CHANGELOG.md

```diff
@@ -9,6 +9,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- `into_base_16_string_no_padding` function to the test helpers to accommodate to Starknet Foundry output where addresses and selectors don't have trailing zeros after `0x` (#1137)
+
+### Changed
+
+- Bump scarb to v2.8.2 (#1137)
+- Bump snforge to 0.30.0 (#1137)
+- `declare_class`, `declare_and_deploy`, and `declare_and_deploy_at` will skip declaration and just return the `ContractClass` if the contract is already declared (#1137)
```

Comment:
`declare_and_deploy`, and `declare_and_deploy_at` don't return the `ContractClass` right? I think we should split this in two different entries.